docs/pages/guides/store-assets.mdx

```diff
@@ -0,0 +1,145 @@
+but you an upload up to 10 screenshots per localisation. Note that if your app is available in multiple languages and your screenshots
+include text, you should upload screenshots with the appropriate language for each localisation.
+
+If your app runs on an iPad, you will need to also provide iPad-sized screenshots.
+
+Screenshots can be portrait or landscape.
+
+### Preview (optional)
+
+An app preview is a video you can choose to include to demonstrate how your app works. You can add up to three app previews for each screen size.
+Reference the [Apple Documentation](https://developer.apple.com/help/app-store-connect/reference/app-preview-specifications/) for a summary on
+video size and format.
+
+## Google Play Store Asset Requirements
+
+Google has specific requirements for the store asset format and dimenstions and they are different from Apple's. For the most up to
+date specifications, reference the [official documentation](https://support.google.com/googleplay/android-developer/answer/9866151) for detailed requirements
+for your Play Store assets.
+
+See our ["Store Asset Requirements" Figma](https://www.figma.com/file/5lQcrHYhUZkIxsLYCZa3S6/Store-Asset-Requirements?type=design&node-id=1-171&mode=design&t=YC6s5lYVJiwSQPki-0) for a summary
+of the minimum asset requirements for an Android App.
+
+### App Icon
+
+Unlike on the iOS App Store where the app icon is always taken automatically from the app bundle,
+on the Google Play store you need to also upload a separate App Icon for your store listing.
+
+### Feature Graphic
+
+A feature graphic must be provided in order to publish your Store Listing. It is a banner that is
+displayed at the top of your store listing page.
+
+### Screenshots
+
+You need to upload at least 4 screenshots in order to publish your app.
+
+### Video (optional)
+
+You can add one preview video to your Store Listing. The video needs to be uploaded to YouTube, and you
+add it by entering a YouTube URL in the "preview video" field.
+
+
+## Bare Mimumim
+
+Below is the bare minimum needed in order to publish your apps.
+
+### App Store - iPhone
+
+| Type                                   | Amount | Dimensions                                | Requirements          |
+|----------------------------------------|--------|-------------------------------------------|-----------------------|
+| Screenshots (iPhone without the notch) | 2-10   | 1242 x 2208                               | JPG or PNG (no alpha) |
+| Screenshots (iPhone with the notch)    | 2-10   | 1290 x 2796 OR 1284 x 2778 OR 1242 x 2688 | JPG or PNG (no alpha) |
+
+### App Store - iPad
+
+If your app also runs on an iPad, you need to supply additional screenshots.
+
+| Type        | Amount | Dimensions  | Requirements          |
+|-------------|--------|-------------|-----------------------|
+| Screenshots | 2-10   | 2048 x 2732 | JPG or PNG (no alpha) |
+
+### Play Store - Android
```

Comment:
Let's also move this section before the iOS ones to follow the platform order guideline.